lib/workers/repository/updates/generate.spec.ts

```diff
@@ -466,7 +466,7 @@ describe('workers/repository/updates/generate', () => {
     });
 
     it('fixes different messages', () => {
-      const branch: BranchUpgradeConfig[] = [
+      const branch: BranchUpgradeConfig[] = partial<BranchUpgradeConfig>([
```

Comment:
```suggestion
      const branch = partial<BranchUpgradeConfig>([
```
do we really need the partial here? isn't satisfies better here?